tests/load/README.md

```diff
@@ -219,6 +208,128 @@ Execute the `setup_k8s.sh` file from the root directory and select the **delete*
 ./tests/load/setup_k8s.sh
 ```
```

Comment:
If we're not demonstrating the `delete` option, do we need to include this `shell` command?

Reply:
fwiw, if nobody told me to do this and if it wasn't documented, I'd have no idea to do it. Given not deleting means a cluster keeps running and it costs us $$, probably doesn't hurt? What do you think?

Reply:
Oh, I'm not saying that it couldn't be useful, just that it's possibly missing something. 

Maybe it's just me, but I tend to read docs like this in one of two ways: 
1) Thoughtful reflection on a new service.
2) Panicked fury because I need something done NOW before the asteroid hits.

Having just this command doesn't really tell me much other than "This is where the file is".

You could change the `read response` to something like 
```bash
response = ${1:-${COMMAND:-$(read r; echo $r)}}
```
which would let you do things like:

`./tests/load/setup_k8s.sh delete` or `COMMAND=delete ./tests/load/setup_k8s.sh` as well as prompt if no value was provided.

Reply:
lol to #2, but also so true.

The read response seems like a good idea for an enhancement of the script